src/scripts/client/modules/gradebook.js

```diff
@@ -136,9 +136,9 @@
                 buildUI( sessionData, $el, gradebook );
             } );
         } catch ( e ) {
-            console.warn( "it blewup", e );
+            console.warn( e );
             // Wrap in try catch to show UI to user that something went wrong ( user permissions? )
-            $el.find( '.tiyo-assistant-notice' ).text( 'There was a problem getting all the data for this gradebooks, do you own this path?!' );
+            $el.text( 'There was a problem getting all the data for this gradebooks, do you own this path?!' );
```

Comment:
As we need to remove some of the pending ui, such as the button that still says pending.   This is a quick way to flush out the structure of the drawer.